libass/ass_bitmap.h

```diff
@@ -62,6 +63,8 @@ typedef void (*ParamFilterFunc)(int16_t *dst, const int16_t *src,
 typedef struct {
     int align_order;  // log2(alignment)
 
+    ASS_CPUFlags cpu_flags;
```

Comment:
I think this should be the first struct member, in the same group as `align_order`, as alignment is directly derived from `cpu_flags`. Maybe it's even better to incorporate `align_order` into `cpu_flags`, i.e. `align_order = cpu_flags & ASS_CPU_ALIGN` or something.

Reply:
Moved to first.